File: TMS/QST.MicroERP.Core/ViewModel/TaskCommentVM.cs
using QST.MicroERP.Core.ViewModel;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QST.MicroERP.Core.ViewModel
{
    public class TaskCommentVM:BaseViewModel
    {
        public int TaskId { get; set; }
        public string? TaskTitle { get; set; }
        public string? User { get; set; }
        public string? UserId { get; set; }
        public string? Comment { get; set; }
        public DateTime Datetime { get; set; }
    }
}

File: TMS/UI/src/app/views/task/models/taskcomment-vm.ts
export class TaskCommentVM {    
    taskId: number
    taskTitle: string
    user: string
    userId: string
    comment: string
    datetime: Date=new Date()
    time: Date=new Date()
    id: number
    isActive: boolean = true;
    createdById: number =0
    createdOn: Date=new Date()
    modifiedById: number
    modifiedOn: Date=new Date()
}